src/components/EmailPasswordInput/EmailPasswordInput.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+
+import { TextInput, TextInputProps, View } from 'react-native';
+import { useController, UseControllerProps } from 'react-hook-form';
+import styles from './styles';
+
+interface FormInputProps extends TextInputProps, UseControllerProps {
```

Comment:
can we rename this to be `EmailPasswordInputProps` to be consistent with the actual component that uses these props? In fact, the component would probably be better off named as something like `AuthInput`